### PR TITLE
New package: InkStitch.InkStitch version 3.0.1

### LINKED ISSUE #115350: [Package Request]: inkstitch
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://inkstitch.org/
Publisher: inkstitch
Package Name: inkstitch
Description: An open source machine embroidery design platform based on Inkscape 
Package Version:  3.0.1
Installer URL:
https://github.com/inkstitch/inkstitch/releases/download/v3.0.1/inkstitch-v3.0.1-windows.exe  
https://github.com/inkstitch/inkstitch/releases/download/v3.0.1/inkstitch-v3.0.1-windows.zip

Requirements:
Ink/Stitch is an Inkscape extension. Depends on Inkscape version 1.0.2 or higher before installing.

### PR DESCRIPTION
**Pull request by Dragon1573**
## Description

- Resolve https://github.com/microsoft/winget-pkgs/issues/115350

## Checks

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/123505)

### CODE REVIEW

**Comment by Dragon1573** [comment]
## Screenshot

Snipaste_2023-10-23_21-11-04

File: manifests/i/InkStitch/InkStitch/3.0.1/InkStitch.InkStitch.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+PackageIdentifier: InkStitch.InkStitch
+PackageVersion: 3.0.1
+PackageLocale: en-US
+Publisher: InkStitch
+PublisherUrl: https://github.com/inkstitch
+PublisherSupportUrl: https://inkstitch.org/docs/faq/
+Author: InkStitch
+PackageName: Ink/Stitch
+PackageUrl: https://github.com/inkstitch/inkstitch
+License: Proprietary
```
Comment (Trenly) [inline manifests/i/InkStitch/InkStitch/3.0.1/InkStitch.InkStitch.locale.en-US.yaml:13]:
```suggestion
License: GPL-3.0
LicenseUrl: https://github.com/inkstitch/inkstitch/blob/main/LICENSE
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
Hi @Dragon1573,

This package installed normally, but did not launch:
image

**Comment by Dragon1573** [comment]
> This package installed normally, but did not launch.

@stephengillie Could you please explain your action on causing this exception? I'll try if I can reproduce it.

**Comment by stephengillie** [comment]
Sure - it was encountered twice:
- Once on automatic program execution.
- Again after manually closing, to confirm the error was legit. (Sometimes automatic execution runs programs in unintended ways.)
It was right after running the program in PowerShell, as it doesn't seem to have its own icon.

**Comment by Dragon1573** [comment]
Sorry, I can't get it and unable to reproduce it ... 😕

> It was right after running the program in PowerShell, as it doesn't seem to have its own icon.

Emmm ... I think it's DESIGNED to have NO icons for itself, and it's DESIGNED to NO launching as a standalone program.

According to [the installation guide](https://inkstitch.org/docs/install-windows/#run-inkstitch), **Ink/Stitch** is an extension of **Inkscape** and it should be used inside Inkscape.

image

Did you try to launch **Ink/Stitch** as a standalone application? I think this is a "false positive" caused by an accidentally incorrect test case.

**Comment by stephengillie** [comment]
Thanks for helping me to find the correct way to launch this application. 
image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**